onboarding/components/step-setup.js

```diff
@@ -0,0 +1,91 @@
+const h = require('react-hyperscript')
+const { compose } = require('recompose')
+const { partial } = require('ramda')
+const { connect: connectStyles } = require('react-fela')
+const { connect: connectStore } = require('redux-bundler-react')
+const { Form, Field } = require('react-final-form')
+const { TextField } = require('redux-form-material-ui')
+const validate = require('redux-form-with-ajv').default
+const Button = require('material-ui/Button').default
+
+const schema = require('../../bots/schemas/createBot')
+const styles = require('../styles/step-setup')
+
+module.exports = compose(
+  connectStyles(styles),
+  partial(connectStore, [
+    'doSubmitOnboardingSetup',
+    'doFindBots',
+    'selectOnboardingUser',
+    'selectOnboardingStepIndex'
+  ])
+)(OnboardingStepSetup)
+
+function OnboardingStepSetup (props) {
+  const { styles } = props
+
+  return h(
+    'div',
+    {
+      className: styles.container
+    },
+    [h(OnboardingStepSetupForm, props)]
+  )
+}
+
+function OnboardingStepSetupForm (props) {
+  const {
+    styles,
+    onboardingUser: user,
+    doSubmitOnboardingSetup: doSubmit
+  } = props
+
+  return h(Form, {
+    onSubmit: doSubmit,
+    initialValues: {
+      userId: user.id
```

Comment:
nah, the userId will be set server-side

Reply:
this makes sense with the above explanation